noxfile.py

```diff
@@ -101,12 +192,242 @@ def performance_test(session: Session) -> None:
     """
     parser = _create_performance_test_arg_parser()
     args = parser.parse_args(session.posargs)
+    file_path = BENCHMARK_FILEPATH / args.file_name
 
     command = [
         "pytest",
         args.test_path,
         "--benchmark-sort=name",
-        f"--benchmark-json={CURRENT_BENCHMARK}",
+        f"--benchmark-json={file_path}",
     ]
-
     session.run(*command)
+
+
+def _create_performance_combine_arg_parser() -> argparse.ArgumentParser:
+    parser = argparse.ArgumentParser(
+        prog="nox -s performance:combine",
+        usage="nox -s performance:test -- [-h] [--path {path}]",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+    parser.add_argument(
+        "--path",
+        dest="path",
+        help="Path specifier for files to combine; should include glob(s) ending with *.json",
+    )
+    return parser
+
+
+@nox.session(name="performance:combine", python=False)
+def performance_combine(session: Session) -> None:
+    """
+    Combine performance benchmarks from multiple JSON files to one
+    file.
+    """
+    parser = _create_performance_combine_arg_parser()
```

Comment:
down-side to combining like this is that it's not the same as the pytest-benchmark generated file, so if we wanted it, we'd need to implement our own histogram function.